pubsub/query/query.peg.go

```diff
@@ -1,3 +1,4 @@
+// nolint
```

Comment:
lol, this is auto generated - maybe you should go run the linter on their repo :P

Reply:
meh